pkg/store/store.go

```diff
@@ -102,3 +102,32 @@ func LoadYAMLByFilePath(filePath string) (*limayaml.LimaYAML, error) {
 	}
 	return y, nil
 }
+
+func Downloads() ([]limayaml.File, error) {
```

Comment:
Probably `limayaml.YAML` object should have `ReferredFiles() ([]limayaml.File, error)` method

Reply:
Ok, let me move it there, np.